docs/source/faq.rst

```diff
@@ -332,3 +332,54 @@ Note that the above examples are similar to running the garbage collector inside
 
     :class:`~optuna.integration.ChainerMNStudy` does currently not provide ``gc_after_trial`` nor callbacks for :func:`~optuna.integration.ChainerMNStudy.optimize`.
     When using this class, you will have to call the garbage collector inside the objective function.
+
+How do I suggest variables which represent the proportion, that is, are in accordance with Dirichlet distribution?
```

Comment:
"flat Dirichlet distribution" (or "uniform Dirichlet distribution") is more appropriate.

Reply:
Thank you for the comment. "flat Dirichlet distribution" sounds more reasonable. I will use it!